gulp/build.js

```diff
@@ -1,41 +1,108 @@
-import gulp from 'gulp';
-import header from 'gulp-header';
-import preprocess from 'gulp-preprocess';
-import rename from 'gulp-rename';
+import gulp       from 'gulp';
+import header     from 'gulp-header';
+import plumber    from 'gulp-plumber';
+import file       from 'gulp-file';
+import filter     from 'gulp-filter';
+import rename     from 'gulp-rename';
```

Comment:
Im not a fan of lining up for those reasons. That said - its a separate
conversation and shouldn't hold up this PR

On Sun, Nov 29, 2015 at 8:27 PM, Paul Falgout <notifications@github.com>
wrote:

> In gulp/build.js
> <https://github.com/marionettejs/backbone.marionette/pull/2834#discussion_r46097473>
> :
>
> > @@ -1,41 +1,108 @@
> > -import gulp from 'gulp';
> > -import header from 'gulp-header';
> > -import preprocess from 'gulp-preprocess';
> > -import rename from 'gulp-rename';
> > +import gulp       from 'gulp';
> > +import header     from 'gulp-header';
> > +import plumber    from 'gulp-plumber';
> > +import file       from 'gulp-file';
> > +import filter     from 'gulp-filter';
> > +import rename     from 'gulp-rename';
>
> We don't. It was done for the imports on the es6 stuff, so this made that
> more consistent. I do think it looks nicer, but it does make it a pain to
> maintain and blame.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/marionettejs/backbone.marionette/pull/2834/files#r46097473>
> .
>